namespaces/_default_.yml

```diff
@@ -187,3 +187,10 @@ _index_type_:
         store or application other than Elasticsearch, but you still need
         to correlate data with the data stored in Elasticsearch, this field
         will give you the exact document corresponding to the record.
+        
+    - name: viaq_index_name
+      type: keyword
+      example: project.my-cool-project-in-lab04.748e92c2-70d7-11e9-b387-000d3af2d83b.2019.05.09
+      description: |
+        Index name in which this message will be stored within the Elasticsearch.
```

Comment:
@richm @jcantrill How is this going to be impacted when moving the new data model with ES 6.x (rollover and common index; no more index per namespace/day).

Reply:
we still need this field

Reply:
https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#data-model